### ISSUE TITLE
Passthrough on overlayPanel is applied multiple times

### ISSUE THREAD

**Issue by cnsgithub** [primefaces/primefaces#1978]
## 1) Environment
- PrimeFaces version: 6.0.12 Elite
- Does it work on the newest PrimeFaces version? No
- Application server + version: WildFly 10.1
- Affected browsers: All

## 2) Expected behavior
Passthrough should only be applied only once (preferably on the ui-overlaypanel)

## 3) Actual behavior
Rendered twice:
<div id="j_idt4:j_idt6" class="ui-overlaypanel ui-widget ui-widget-content ui-overlay-hidden ui-corner-all ui-shadow" onclick="alert('twice?');"><div class="ui-overlaypanel-content" onclick="alert('twice?');">

## 4) Steps to reproduce
n/a

## 5) Sample XHTML

```

<html xmlns="http://www.w3.org/1999/xhtml" xmlns:h="http://java.sun.com/jsf/html" xmlns:p="http://primefaces.org/ui" xmlns:pt="http://xmlns.jcp.org/jsf/passthrough">  
	<h:head>
		<title>OverlayPanel passthrough Bug</title>
	</h:head>
	<h:form>
	<p:commandButton value="Foo" type="button" />
	<p:overlayPanel for="@previous" pt:onclick="alert('twice?');">
	        <div>foo</div>
    </p:overlayPanel>
    
	</h:form>
</html>  

```

## 6) Sample bean
No bean needed, reproducible only by using XHTML

Edit: fileUpload is affected too - here 3 times

**Comment by tandraschko** [comment]
Feel free to provied a PR to fix this problem.

**Comment by EnmanuelReyes** [comment]
@cnsgithub can you provide a sample view and a sample bean for the fileUpload component?
I already resolved the issue with the OverlayPanel and I don't want to make a PR only for fix the OverlayPanel knowing that the fileUpload has the same bug.

**Comment by tandraschko** [comment]
We only accept one PR for one issue, so please don't mix them together.

**Comment by tandraschko** [comment]
Applied for Overlay panel now. Thanks for the PR.
If the same is still required for FileUpload, please create a new issue.
